vertical-pod-autoscaler/e2e/v1/full_vpa.go

```diff
@@ -43,7 +43,7 @@ const (
 	// the initial values should be outside minimal bounds
 	initialCPU     = int64(10) // mCPU
 	initialMemory  = int64(10) // MB
-	oomTestTimeout = 12 * time.Minute
+	oomTestTimeout = 20 * time.Minute
```

Comment:
This change worries me - the point of OOM test is that it should be quicker than scaling without OOMs.

With this we expect them to be slower. `pollTmeout` is 15 minuutes ([v1beta2](https://github.com/kubernetes/autoscaler/blob/04e5bfc88363b4af9fdeb9dfd06c362ec5831f51/vertical-pod-autoscaler/e2e/v1beta2/common.go#L51), [v1](https://github.com/kubernetes/autoscaler/blob/04e5bfc88363b4af9fdeb9dfd06c362ec5831f51/vertical-pod-autoscaler/e2e/v1/common.go#L51)).

Can you explain why we need to increase this timeout?

Reply:
Reverted back. Thanks!